build-recipe-livebuild

```diff
@@ -193,9 +193,9 @@ recipe_build_livebuild() {
 
     if [ -x $BUILD_ROOT/usr/lib/build/livebuild_pre_run ] ; then
 	echo "Running OBS build livebuild_pre_run hook"
-	chroot $BUILD_ROOT su -c \
+	chroot $BUILD_ROOT sh -c \
         "export RECIPEFILE=${RECIPEFILE}; /usr/lib/build/livebuild_pre_run" \
```

Comment:
same here

Reply:
and in the next lines;)

Reply:
Here and below it's running a shell line, not a single command (eg, export or change directory, then run thing), so I don't think I can for this one

Reply:
right